plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/api/command/LdapListUsersCmd.java

```diff
@@ -38,7 +47,35 @@
 
 import com.cloud.user.Account;
 
-@APICommand(name = "listLdapUsers", responseObject = LdapUserResponse.class, description = "Lists all LDAP Users", since = "4.2.0",
+/**
+ * @startuml
```

Comment:
How to read, make sense out of this? Are you adding some new doc/style?

Reply:
plantuml, do you want it removed? it is an explaining flow chart/state diagram. I could add a comment explaining, but that would have to go anywhere plant is used